e2e/pages/Browser/TestSnap.js

```diff
@@ -0,0 +1,874 @@
+import enContent from '../../../locales/languages/en.json';
+import Browser from './BrowserView';
+import Matchers from '../../utils/Matchers';
+import { BrowserViewSelectorsIDs } from '../../selectors/Browser/BrowserView.selectors';
+import { 
+  TestSnapViewSelectorWebIDS, 
+  TestSnapInputSelectorWebIDS, 
+  TestSnapResultSelectorWebIDS, 
+  TestSnapDropdownSelectorWebIDS 
+} from '../../selectors/Browser/TestSnap.selectors';
+import Gestures from '../../utils/Gestures';
+import { SNAP_INSTALL_CANCEL, SNAP_INSTALL_CONNECT, SNAP_INSTALL_CONNECTION_REQUEST } from '../../../app/components/Approvals/InstallSnapApproval/components/InstallSnapConnectionRequest/InstallSnapConnectionRequest.constants';
+import { SNAP_INSTALL_PERMISSIONS_REQUEST, SNAP_INSTALL_PERMISSIONS_REQUEST_APPROVE } from '../../../app/components/Approvals/InstallSnapApproval/components/InstallSnapPermissionsRequest/InstallSnapPermissionsRequest.constants';
+import { SNAP_INSTALL_OK } from '../../../app/components/Approvals/InstallSnapApproval/InstallSnapApproval.constants';
+import { web, by } from 'detox';
+
+export const TEST_SNAP_URL = 'https://metamask.github.io/snaps/test-snaps/latest/';
```

Comment:
```suggestion
export const TEST_SNAPS_URL = 'https://metamask.github.io/snaps/test-snaps/latest/';
```

Reply:
This should point to a fixed version:
```suggestion
export const TEST_SNAPS_URL = 'https://metamask.github.io/snaps/test-snaps/2.23.1/';
```